src/components/App/index.tsx

```diff
@@ -37,8 +37,15 @@ export const App: FC = () => {
     if (!!whoAmI) {
       dispatch({ type: SET_USER_DATA, payload: whoAmI });
     }
-    if (whoAmI?.courses[0])
-      dispatch({ type: SET_CURR_COURSE, payload: whoAmI?.courses[0] });
+    if (whoAmI?.courses[0]) {
+      if (!localStorage.getItem('currCourse')) {
+        localStorage.setItem('currCourse', JSON.stringify(whoAmI?.courses[0]));
+      }
+      const currCourse = JSON.parse(
+        localStorage.getItem('currCourse') as string
```

Comment:
Could you save 'currCourse' as a const and use it where it's necessary?

Reply:
Done